aqt/archives.py

```diff
@@ -371,15 +408,15 @@ def _parse_update_xml(self, archive_url, target_packages):
                     downloadable_archives = _archives.split(", ")
                 else:
                     downloadable_archives = []
-                full_version = packageupdate.find("Version").text
-                if not full_version.startswith(self.version):
+                named_version = packageupdate.find("Version").text
+                full_version = Version(named_version)
+                if not full_version.base_version == self.version.base_version:
                     self.logger.warning(
-                        "Version {} differ from requested version {} -- skip.".format(
-                            full_version, self.version
+                        "Base Version of {} is differed from requested version {} -- skip.".format(
```

Comment:
I think you mean `is different` instead of `is differed`.

Reply:
yes.